src/error.rs

```diff
@@ -94,6 +94,33 @@ pub enum BDKCliError {
     #[cfg(feature = "cbf")]
     #[error("BDK-Kyoto update error: {0}")]
     KyotoUpdateError(#[from] bdk_kyoto::UpdateError),
+
+    #[error("Mnemonic generation failed: {0}")]
+    MnemonicGenerationError(String),
+
+    #[error("Xpriv creation failed: {0}")]
+    XprivCreationError(String),
+
+    #[error("Descriptor parsing failed: {0}")]
+    DescriptorParsingError(String),
+
+    #[error("Invalid extended public key (xpub): {0}")]
+    InvalidXpub(String),
+
+    #[error("Invalid extended private key (xprv): {0}")]
+    InvalidXprv(String),
+
+    #[error("Invalid derivation path: {0}")]
+    InvalidDerivationPath(String),
+
+    #[error("Unsupported script type: {0}")]
+    UnsupportedScriptType(u8),
+
+    #[error("Descriptor key conversion failed: {0}")]
+    DescriptorKeyError(String),
+
+    #[error("Invalid arguments: {0}")]
+    InvalidArguments(String),
```

Comment:
This variant seems to be duplicating the Generic Variant. See output below:

<img width="799" height="123" alt="Screenshot 2025-08-19 at 6 21 45 PM" src="https://github.com/user-attachments/assets/7017bec4-a528-4441-b8ae-1cabcc679452" />